src/data/layer-2-data-files/optimistic-rollups.json

```
@@ -0,0 +1,30 @@
+[
+  {
+    "name": "Arbitrum",
+    "website": "https://arbitrum.io/",
+    "developerDocs": "https://developer.offchainlabs.com/docs/developer_quickstart",
+    "l2beat": "https://l2beat.com/projects/arbitrum/",
+    "note": ""
+  },
+  {
+    "name": "Metis Andromeda",
+    "website": "https://metis.io",
+    "developerDocs": "https://docs.metis.io/",
+    "l2beat": "https://l2beat.com/projects/metis/",
+    "note": "Fraud proofs in development"
```

These notes should be translated too

```suggestion
    "note": "Fraud proofs in development"
```

---

Yeah agree, we should extract these for translation. I suggested changing the key to `"noteKey"` to remind us this should be a lookup key, and keep the string in the language json.

The name and websites of course don't need to be translated.

---

Sounds good, thanks for suggestion. Will get this done in next commit, just wanted to get this in an acceptable place before wrapping that up.